### PR TITLE
Fix humphd/brackets#206 - MVP of a Mobile Preview mode

### PR DESCRIPTION
**Pull request by AJDBenner**
Fix https://github.com/humphd/brackets/issues/206.

Screenshots

Before
screen shot 2015-04-20 at 12 52 47 pm

After
screen shot 2015-04-20 at 12 52 58 pm

### CODE REVIEW

File: lib/Mobile.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<style type="text/css" id="phoneStyles">
```
Comment (humphd) [inline lib/Mobile.html]: Move all CSS to the external file.

**Comment by humphd** [comment]
This is looking really good.  Can we get some padding above the top of the phone so it doesn't sit right against the top Preview bar?  Maybe 20-30px or something?  Or do we want to consider centering this vertically too?

**Comment by sedge** [comment]
Looks good to me too. Vertical centering _would_ be nice

**Comment by gideonthomas** [comment]
vertical centering assigned to @sedge :)

**Comment by AJDBenner** [comment]
Updated screen shot
screen shot 2015-04-20 at 1 27 50 pm

**Comment by humphd** [comment]
I think this is r+ and we can fix things on it in new bugs.  Well done!

@sedge can you r? this too?

**Comment by sedge** [comment]
I like what I see. R+

**Pull request merged**